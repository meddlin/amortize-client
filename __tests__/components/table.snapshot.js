import { render } from '@testing-library/react';
import Table from '../../src/components/table';

it('renders a table', () => {
    const { container } = render(<Table />)
    expect(container).toMatchSnapshot()
})