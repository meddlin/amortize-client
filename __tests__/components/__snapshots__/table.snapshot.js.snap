// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a table 1`] = `
<div>
  <div
    class="px-4 sm:px-6 lg:px-8"
  >
    <div
      class="sm:flex sm:items-center"
    >
      <div
        class="sm:flex-auto"
      >
        <h1
          class="text-base font-semibold leading-6 text-gray-900"
        >
          Amortization
        </h1>
        <p
          class="mt-2 text-sm text-gray-700"
        >
          See the calculated amortization table below.
        </p>
      </div>
      <div
        class="mt-4 sm:ml-16 sm:mt-0 sm:flex-none"
      >
        <button
          class="block rounded-md bg-indigo-600 px-3 py-2 text-center text-sm font-semibold text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600"
          type="button"
        >
          Some button
        </button>
      </div>
    </div>
    <div
      class="mt-8 flow-root"
    >
      <div
        class="-mx-4 -my-2 overflow-x-auto sm:-mx-6 lg:-mx-8"
      >
        <div
          class="inline-block min-w-full py-2 align-middle sm:px-6 lg:px-8"
        >
          <table
            class="min-w-full divide-y divide-gray-300"
          >
            <thead>
              <tr>
                <th
                  class="px-3 py-3.5 text-left text-sm font-semibold text-gray-900"
                >
                  <div>
                    <h3>
                      Term
                    </h3>
                  </div>
                </th>
                <th
                  class="px-3 py-3.5 text-left text-sm font-semibold text-gray-900"
                >
                  <div>
                    <h3>
                      Monthly Payments
                    </h3>
                  </div>
                </th>
                <th
                  class="px-3 py-3.5 text-left text-sm font-semibold text-gray-900"
                >
                  <div>
                    <h3>
                      Principal
                    </h3>
                  </div>
                </th>
                <th
                  class="px-3 py-3.5 text-left text-sm font-semibold text-gray-900"
                >
                  <div>
                    <h3>
                      Interest
                    </h3>
                  </div>
                </th>
                <th
                  class="px-3 py-3.5 text-left text-sm font-semibold text-gray-900"
                >
                  <div>
                    <h3>
                      Rem. Principal
                    </h3>
                  </div>
                </th>
                <th
                  class="px-3 py-3.5 text-left text-sm font-semibold text-gray-900"
                >
                  <div>
                    <h3>
                      Extra Payment
                    </h3>
                  </div>
                </th>
              </tr>
            </thead>
            <tbody
              class="divide-y divide-gray-200"
            >
              <tr>
                <td
                  class="whitespace-nowrap px-3 py-4 text-sm text-gray-500"
                >
                  Data here
                </td>
              </tr>
            </tbody>
          </table>
        </div>
      </div>
    </div>
  </div>
</div>
`;
